spec/factories/fields.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :field do
+    calculator_id { 1 }
```

Comment:
you should use an association here

Reply:
Thanks. I've taken this into account in my last commit.